netdata-installer.sh

```diff
@@ -254,7 +255,11 @@ while [ -n "${1}" ]; do
     "--disable-x86-sse") NETDATA_CONFIGURE_OPTIONS="${NETDATA_CONFIGURE_OPTIONS//--disable-x86-sse/} --disable-x86-sse" ;;
     "--disable-telemetry") NETDATA_DISABLE_TELEMETRY=1 ;;
     "--disable-go") NETDATA_DISABLE_GO=1 ;;
-    "--disable-cloud") NETDATA_DISABLE_LIBMOSQUITTO=1 ; NETDATA_CONFIGURE_OPTIONS="${NETDATA_CONFIGURE_OPTIONS//--disable-aclk/} --disable-aclk" ;;
+    "--enable-ebpf") NETDATA_ENABLE_EBPF=1 ;;
```

Comment:
If i remember it right we will have several ebpf plugins. Should this flag be more specific? Or the plan is either to download all ebpf stuff (several plugins) or nothing?

Reply:
Not entirely sure to be honest. I think @thiagoftsm has more knowledge around this. My gut instinct tells me we ship all ebpf programs.

Reply:
You are right @ilyam8 , and I am using  on 7979 `NETDATA_ENABLE_EBPF' because the libraries to enable one is the exact the same to enable others.